armory/datasets/standard/carla_obj_det_train/carla_obj_det_train_test.py

```diff
@@ -0,0 +1,25 @@
+"""carla_obj_det_train dataset."""
```

Comment:
is this the file you were asking about? You can look [here](https://github.com/twosixlabs/armory/tree/tfdsv4/armory/datasets/standard) for examples of what already-integrated datasets look like. We should have tests for this dataset already in `tests/`, so this file along with `TODO-add_fake_data...` I believe can be removed

Reply:
Resolved with 941be98